src/CarbonAware.CLI/test/integrationTests/Commands/EmissionsForecasts/EmissionsForecastsCommandTests.cs

```diff
@@ -10,6 +10,7 @@ namespace CarbonAware.CLI.IntegrationTests.Commands.EmissionsForecasts;
 /// </summary>
 [TestFixture(DataSourceType.WattTime)]
 [TestFixture(DataSourceType.ElectricityMaps)]
+[TestFixture(DataSourceType.ElectricityMapsFree)]
```

Comment:
Remove this because it seems `ElectricityMapsFree` doesn't support any of the paths that this tests except for the helptext path, and that path is tested with all the other ones too. And then you can remove all the `IgnoreTestForDataSource` lines in all the functions below